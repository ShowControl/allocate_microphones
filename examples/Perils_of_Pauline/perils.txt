[show The Perils of Pauline]  // name the show

[character Pauline]
[character Harry Marvin]
[character Raymond Owen]

[actor Pearl White]
[actor Crane Wilbur]

[role]
actor=Pearl White
character=Pauline

[role]
actor=Crane Wilbur
character=Harry Marvin

[page 1]
[act 1]
[scene 1]
[enter Pauline]
[song Where is My Hero]
[page 2]
[enter Harry Marvin]
[song I Have Come to Save You]
[page 3]
[exit Harry Marvin]
[enter Raymond Owen]
[begin sound Train Approaching]
[page 4]
[enter Harry Marvin]
[quiet Raymond Owen]
[page 5]
[end sound Train Approaching]
[exit Raymond Owen]
[page 6]
[exit Pauline]
[exit Harry Marvin]
[end show]

