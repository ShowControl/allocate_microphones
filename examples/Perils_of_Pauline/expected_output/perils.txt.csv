page, "Pearl White", "Crane Wilbur", "Uncast #1"
"1","*"," "," "
"2","*","*"," "
"3","*"," ","*"
"4","*","*","*"
"5","*","*"," "
"6"," "," "," "
